.travis.yml

```diff
@@ -3,7 +3,7 @@ languages: java
 jdk:
   - oraclejdk8
 sudo: false
-script: ./gradlew clean build
+script: ./gradlew check
```

Comment:
I am a bit skeptical about this. We have a lot of distribution code where we package stuff from a ton of locations. Will it be a good idea to stop exercising this code totally? I am wondering whether we could potentially run into surprises while publishing.

IMO, I would sleep better if I know that all targets build and are working fine. I can sacrifice a minute of build time for that. Let me know your thoughts.